homeassistant/components/sensor/homematicip_cloud.py

```diff
@@ -177,30 +145,22 @@ def unit_of_measurement(self):
     def device_state_attributes(self):
         """Return the state attributes."""
         return {
-            ATTR_VALVE_STATE: self._device.valveState.lower(),
             ATTR_TEMPERATURE_OFFSET: self._device.temperatureOffset,
             ATTR_LOW_BATTERY: self._device.lowBat,
```

Comment:
Isn't this already returned in the parent class `device_state_attributes`? You could fetch those from the parent and then add the child specific attributes to the dict.

```py
attr = super().device_state_attributes
attr.update(...)
return attr
```

Reply:
I have further reduced state attributes. I have seen somewhere also the discussion to general add a attribute for the battery state which I would highly appreciate.

Next I would like more components and most important a configuration GUI to generate the authentication token.

Later would like to introduce something like a DEBUG switch in the configuration where interested users can have some of the attributes in a dedicated sensor back.